src/components/Header/index.js

```diff
@@ -24,11 +20,7 @@ function HeaderIndex() {
     <>
       <div className="header">
         <Link to="/">
-          <img
-            src="https://cdn.worldvectorlogo.com/logos/arrow-2.svg"
-            alt="logo"
-            className="header__logo"
-          />
+          <img src={LOGO_IMG} alt="logo" className="header__logo" />
```

Comment:
Try to create a component called `Logo` that always renders the logo.

Reply:
okay, will do it